samples/sensor/adxl372/prj.conf

```diff
@@ -2,7 +2,6 @@ CONFIG_STDOUT_CONSOLE=y
 CONFIG_LOG=y
 CONFIG_SPI=y
 CONFIG_SENSOR=y
-CONFIG_ADXL372=y
 CONFIG_ADXL372_SPI=y
```

Comment:
btw. we should also get rid of those kconfig symbols and use some macros to determine the bus but that would be a case for another PR.

Reply:
We can rework the driver to not need the Kconfig symbols and do what other drivers do:
`DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)` and `DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)`